packages/main/src/ComboBox.ts

```diff
@@ -624,37 +671,37 @@ class ComboBox extends UI5Element {
 		});
 	}
 
-	handleNavKeyPress(event) {
-		if (this.focused && (isHome(event) || isEnd(event)) && this.value) {
+	handleNavKeyPress(e: KeyboardEvent) {
+		if (this.focused && (isHome(e) || isEnd(e)) && this.value) {
 			return;
 		}
 
 		const isOpen = this.open;
 		const currentItem = this._filteredItems.find(item => {
-			return isOpen ? item.focused : item.selected;
-		});
+			return isOpen ? item.focused : (item as ComboBoxItem).selected;
+		})!;
 		const indexOfItem = this._filteredItems.indexOf(currentItem);
 
-		event.preventDefault();
+		e.preventDefault();
 
-		if (this.focused && isOpen && (isUp(event) || isPageUp(event) || isPageDown(event))) {
+		if (this.focused && isOpen && (isUp(e) || isPageUp(e) || isPageDown(e))) {
 			return;
 		}
 
-		if (this._filteredItems.length - 1 === indexOfItem && isDown(event)) {
+		if (this._filteredItems.length - 1 === indexOfItem && isDown(e)) {
 			return;
 		}
 
 		this._isKeyNavigation = true;
 
-		this[`_handle${event.key}`](event, indexOfItem);
+		(this as any)[`_handle${e.key}`](event, indexOfItem);
```

Comment:
If you want to keep it, the way to get rid of "any" is to provide condition, for example to ensure the method exists on the this, because e.key can be a lot of things

```ts
// you can of course make a white list and simplify the condition expression, but this is just for the sake of the explanation
if (e.key === "ArrowDown" || e.key === "ArrowUp" || e.key === "PageUp" || e.key === "PageDown") {
			this[`_handle${e.key}`](e, indexOfItem);
		}
```